scoresheets/Receptionist.tex

```diff
@@ -1,14 +1,13 @@
-The maximum time for this test is \textbf{5 minutes}.
+The maximum time for this test is 5 minutes.
 
-\begin{scorelist}
+\begin{scorelist}[startbutton=false]
 	\scoreheading{Main Goal}
-	\scoreitem{100}{Introduce the 1st guest to John}
-	\scoreitem{150}{Offer an empty seat to the 1st guest}
-	\scoreitem{100}{Introduce the 2nd guest to John and 1st guest}
-	\scoreitem{150}{Offer an empty seat to the 2nd guest}
-	\penaltyitem[2]{50}{Misunderstanding the name of a guest}
+	\scoreitem[2]{250}{Introduce a new guest to every other guest and offer a seat}
+	\penaltyitem[2]{50}{Continue with wrong name or drink}
+	\penaltyitem[2]{75}{Alternative HRI}
+	\penaltyitem[2]{200}{Recognizing People}
```

Comment:
This item should be named more similarly to the storing groceries DeM penalties; at first glance it looks like recognizing people somehow causes a penalty. "Request person recognition"

Reply:
Changed